### DISCUSSION TITLE
Clipboard plugin intermittent .NET crash

### DISCUSSION THREAD

**Discussion by deefrawley** [Flow-Launcher/Flow.Launcher#559] [Q&A] [answered]
I couldn't find a plugin repo and I know this was a Wox port - should I just post the crash as a Flow issue?

**Comment by taooceros** [comment]
Does it crash Flow?

**Comment by taooceros** [comment, marked as answer]
The repo: https://github.com/Xenolphthalein/Flow.Launcher.Plugin.ClipboardHistory

**Reply by deefrawley** [reply]
Thanks
